exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/ViewHandler.java

```diff
@@ -62,9 +62,10 @@ public PhysicalPlan getPlan(SqlNode sqlNode) throws ValidationException, RelConv
 
       final String newViewName = createView.getName();
 
+      // Disallow temporary tables usage in view definition
+      config.getConverter().disallowTemporaryTables();
```

Comment:
Looks like we only disable temporary tables in create view flow? What about view expansion logic? 

Let's say under "dfs.tmp", I have a temporary table "t1" which is masked into what_ever_random_name, and I also have folder "t1" for a regular table "t1".  When we create a view on top of "t1", since temporary table is disabled, we will resolve to this regular table "t1". However, when we run a query against a view, do we resolve to the temporary table in stead? Is this what we expect to see?